upup/models/cloudup/resources/addons/networking.weave/k8s-1.12.yaml.template

```diff
@@ -125,20 +123,17 @@ metadata:
   name: weave-net
   labels:
     name: weave-net
-    role.kubernetes.io/networking: "1"
   namespace: kube-system
 spec:
   # Wait 5 seconds to let pod connect before rolling next pod
   selector:
     matchLabels:
       name: weave-net
-      role.kubernetes.io/networking: "1"
```

Comment:
@hakman  this breaks the weave addon update as selector labels are immutable, logs from protokube 

```
The DaemonSet "weave-net" is invalid: spec.selector: Invalid value: v1.LabelSelector{MatchLabels:map[string]string{"name":"weave-net"}, MatchExpressions:[]v1.LabelSelectorRequirement(nil)}: field is immutable

error updating "networking.weave": error applying update from "s3://kops...": error running kubectl: exit status 1
```

Reply:
Support for weave has been removed now due to the inactivity of upstream. So I while we did an unintentional breaking change here, I think the path forward is to migrate to another CNI.

Reply:
Thanks for the reply, migration from weave is planned but this added an extra twist.